src/metrics_classification.rs

```diff
@@ -39,6 +39,7 @@ fn map_prediction_to_idx<L: Label>(
 /// A confusion matrix shows predictions in a matrix, where rows correspond to target and columns
 /// to predicted. Diagonal entries are correct predictions, and everything off the
 /// diagonal is a miss-classification.
+#[derive(Clone, PartialEq)]
```

Comment:
`Debug` derive here

Reply:
It's implemented from the line 242. Although I would propose to change it to Display, and add derived Debug

Reply:
If a `Debug` impl already exists then we should just leave it as it, no need to add `Display`.